.script/documentsLinkValidator.ts

```diff
@@ -0,0 +1,27 @@
+import { runCheckOverChangedFiles } from "./utils/changedFilesValidator";
+import { ExitCode } from "./utils/exitCode";
+import fs from "fs";
+import * as logger from "./utils/logger";
+
+export async function IsFileContainsLinkWithLocale(filePath: string): Promise<ExitCode> {
+  const content = fs.readFileSync(filePath, "utf8");
+  if (/(https:\/\/docs.microsoft.com|https:\/\/azure.microsoft.com)(\/[a-z]{2}-[a-z]{2})/i.test(content)) {
+      throw new Error();
```

Comment:
Maybe add custom error message here?
I think this message will be displayed in `onExecError`  by `{e.message}`